target/linux/ath79/dts/qca9563_kuwfi_wb5g08.dts

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca956x.dtsi"
+
+#include <dt-bindings/leds/common.h>
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "kuwfi,wb5g08", "qca,qca9563";
+	model = "KuWfi WB5G08";
+
+	aliases {
+		label-mac-device = &wmac;
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_system: power {
+			function = LED_FUNCTION_POWER;
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+
+		wlan {
+			function = LED_FUNCTION_WLAN;
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio 12 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		link1 {
+			label = "green:signal0";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio 1 GPIO_ACTIVE_HIGH>;
+		};
+
+		link2 {
+			label = "green:signal1";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+		};
+
+		link3 {
+			label = "green:signal2";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio 3 GPIO_ACTIVE_LOW>;
+		};
+
+		link4 {
+			label = "green:signal3";
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&gpio 0 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {  
+                compatible = "gpio-keys";
+
+                reset { 
+                        label = "Reset button";
+                        linux,code = <KEY_RESTART>;
+                        gpios = <&gpio 5 GPIO_ACTIVE_LOW>;
+                        debounce-interval = <60>;
+                };
+        };
+};
+
+&pcie {
+        status = "okay";
+
+        wifi@0,0 {
+                compatible = "qcom,ath10k";
+                reg = <0x0000 0 0 0 0>;
+                qcom,ath10k-calibration-variant = "KuWfi-WB5G08";
+                nvmem-cells = <&cal_art_1000>,<&cal_art_5000>;
+                nvmem-cell-names = "pre-calibration","calibration";
```

Comment:
You can not add both of them.

Reply:
This is this only way to make it works, this is also what the original OEM does

Reply:
Last combination that I have not tried yet also works

nvmem-cells = <&cal_art_5000>;
nvmem-cell-names = "pre-calibration";